src/arrays.js

```diff
@@ -1,77 +1,94 @@
 const getNthElement = (index, array) => {
-  // your code here
+    if (index < array.length) {
```

Comment:
One thing you might want to look at here which is pretty cool, is the [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator). You can use it in scenarios like this one to cut down on if/else statements:

```js
return (index < array.length) 
  ? array[index]
  : array[index - array.length] 
```